setup.cfg

```diff
@@ -3,7 +3,7 @@ format = pylint
 exclude = .svc,CVS,.bzr,.hg,.git,__pycache__,venv,venv3,node_modules
 max-complexity = 10
 max-line-length = 120
-ignore = NONE
+ignore = W503
```

Comment:
just found this, why do we ignore this one?

Reply:
Both W503 and W504 are enabled by default and they contradict one another. The error raised in W503 is planned to be best practice by Flake 8. Therefore we should ignore it. 
 
See these links
- https://gitlab.com/pycqa/flake8/-/issues/466
- https://stackoverflow.com/questions/57074300/what-is-the-recommended-way-to-break-long-if-statement-w504-line-break-after-b

Reply:
i c